installation_guide.md

```diff
@@ -0,0 +1,237 @@
+
+# PySyft Installation Guide
+
+Before you start following the tutorials, you need to install PySyft on your computer.
+
+Here are the PySyft installation instructions for Linux, MacOs and Windows.
+
+Just scroll down to find your preferred OS:
+
+--------------
+
+Authors:
+
+- Alan Aboudib - Twitter: [@alan_aboudib](https://twitter.com/alan_aboudib)
+
+
+
+----------------------------
+## 0.1 Linux
+--------------------
+
+Installing PySyft on Linux is really straight forward. Here are the steps:
+
+#### 1. Make sure you have python >= 3.6
+
+You can check out the version number by running:
+
+<font color='red'>$</font>  `python --version`
+
+#### 2. Install pytorch 1.0:
```

Comment:
This isnt need, setup.py takes care of it

Reply:
handled!